Update the "Customizing the HTML5 Shell" page (#4748)
godotengine/godot-docs: tutorials/platform/customizing_html5_shell.rst

  The name is generated from the *Export Path* during the export process.

- ``$GODOT_CONFIG``:
  A JavaScript object, containing the export options and can be later overridden.
  See :js:attr:`EngineConfig` for the full list of overrides.

The following optional placeholders will enable some extra features in your cusstom HTML template.

- ``$GODOT_PROJECT_NAME``:
  The project name as defined in the Project Settings. It is a good idea to use it as a ``<title>``
  in your template.

- ``$GODOT_HEAD_INCLUDE``:
  A custom string to include in the HTML document just before the end of the ``<head>`` tag. It
  is customized in the export options under the *Html / Head Include* section. While you fully
  control the HTML page you create, this variable can be useful for configuring parts of the
  HTML ``head`` element from the Godot Editor, e.g. for different Web export presets.

When the custom page is ready, it can be selected in the export options under the *Html / Custom Html Shell*
section.

.. image:: img/html5_export_options.png

Starting the project
--------------------
To be able to start the game, you need to write a script that initializes the engine — the control

> code. This process consists of three steps, though as shown most of them can be skipped depending on

Reviewer: ```suggestion
code. This process consists of three steps, but as shown here, most of them can be skipped depending on
```

> how much customization is needed (or be left to a default behavior).

Reviewer: ```suggestion
how much customization is needed.
```

``console.log`` and ``console.warn`` are used for the output and error streams respectively. This
behavior can be customized by setting your own functions to handle messages.

Use the :js:attr:`onPrint` override option to set a callback function for the output stream,
and the :js:attr:`onPrintError` override option to set a callback function for the error stream.

.. code-block:: js

    function print(text) {
        console.log(text);
    }
    function printError(text) {
        console.warn(text);
    }
    engine.startGame({ onPrint: print, onPrintError: printError });

> When handling the engine output keep in mind, that it may not be desirable to print it out in the

Reviewer: ```suggestion
When handling the engine output, keep in mind that it may not be desirable to print it out in the
```
